solution-base/kafka/charts/kafka-operator/templates/operator-validating-webhook.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if and (.Values.webhook.enabled) (.Values.webhook.certs.generate) -}}
+
+{{- $commonName := printf "%s-operator.%s.svc" (include "kafka-operator.fullname" .) .Release.Namespace -}}
+---
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
+metadata:
+  name: {{ include "kafka-operator.name" . }}-issuer
+  namespace: {{ .Release.Namespace }}
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
should we set this to `cert-manager.io/v1` now?

Reply:
So here are some thoughts of mine, if we start patching the chart, I would rather track that for the day we need to upgrade the upstream. either with some patch files.

Or actually, make a pr on the real kafka operator chart to `v1`

Reply:
`.patch` files would definitely help.
the webhook cert-manager resource template does not exist in `kafka-operator` .

Reply:
oh if it doesn't exist, in that case we can do whatever we want with this file